### PR TITLE
Sysctl proc sys kernel

### PR DESCRIPTION
**Pull request by majuscule**
PR https://github.com/duckduckgo/zeroclickinfo-fathead/pull/79 with updated reference branch. @jdorweiler 

---
https://duck.co/ia/view/sysctl

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Fathead/SysctlProcSysKernel.pm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package DDG::Fathead::SysctlProcSysKernel;
```
Comment (jdorweiler) [inline lib/DDG/Fathead/SysctlProcSysKernel.pm]: You can remove this file we don't need it anymore.

**Comment by jdorweiler** [comment]
Can you move your files to `/lib/fathead/sysctl/`?

**Comment by majuscule** [comment]
Done, and rebased onto the latest master, apologies for the history rewrite but this should be nicer :-)

**Comment by jdorweiler** [comment]
Looks pretty good to me.  I deployed it to beta, want to check it out? It just triggers on `linux` right now.
https://beta.duckduckgo.com/?q=linux+panic_on_stackoverflow&ia=about

**Comment by majuscule** [comment]
Is the singular `linux` trigger an artifact of the beta deploy or would that be the case in production? It should trigger for all of the examples given in the meta.txt file. In addition, it should trigger on direct searches in the form `/proc/sys/kernel/hung_task_check_count`, which may have been an oversight by me? That form should probably be the canonical resource that the rest redirect to.

**Comment by jdorweiler** [comment]
> Is the singular linux trigger an artifact of the beta deploy or would that be the case in production? It should trigger for all of the examples given in the meta.txt file. 

Yes, just for the beta deploy.  I'll add the rest of the triggers once we're happy with the content. 

>In addition, it should trigger on direct searches in the form /proc/sys/kernel/hung_task_check_count, which may have been an oversight by me? That form should probably be the canonical resource that the rest redirect to.

You would need to add redirect entries for this.  Something like:
```
/proc/sys/kernel/hung_task_check_count    R    hung_task_check_count
```

That should work.  Although I'm not sure of any other fatheads that have keys with a `/` in them but it doesn't hurt to try :smile:

**Comment by moollaza** [comment]
Just checking in here guys -- are we waiting on code changes here or a beta deploy?

**Comment by moollaza** [comment]
I know @jdorweiler is away this week so Beta deploy will have to be next week

**Comment by majuscule** [comment]
Best would be to fix the redirect rules as I mentioned above, but I think this is already functional and ready to go. As I won't have time to update the code for that feature anytime too soon, I think we should proceed.

**Comment by edgesince84** [comment]
@majuscule hey, I wanted to see if you still planned on making any more changes.  It's been a bit since we addressed this and I want to see this go live.  cc @jdorweiler @moollaza 

If you're interested in working on something new, we'd like to invite you to our [new forum](https://forum.duckduckhack.com/) to preview how we're improving programming-related searches. There are a lot of important tasks up for grabs that we'd like your help with, so feel free to and [jump in](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).

**Comment by moollaza** [comment]
@majuscule @edgesince84 now that we can deploy Fatheads to Beta more easily, I've deployed this PR: 

- https://beta.duckduckgo.com/?q=bootloader_version&ia=about
- https://beta.duckduckgo.com/?q=panic_on_unrecovered_nmi&ia=about

@jdorweiler any feedback?

**Comment by jdorweiler** [comment]
Looks good @majuscule. Sorry this took so long.

**Pull request merged**